packages/transaction-controller/src/types.ts

```diff
@@ -82,6 +82,16 @@ type TransactionMetaBase = {
    */
   chainId: Hex;
 
+  /**
+   * A string representing a name of transaction contract method.
+   */
+  contractMethodName?: string;
```

Comment:
I can see we're now setting `originalType`, but do we know where the other new properties are used?

If they're not used in the core `TransactionController`, do we either need to add the logic that sets them or can we ignore them and remove them from the extension also?

Reply:
The other new properties are set on [token-allowance.js](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/token-allowance/token-allowance.js#L263) and [updateTxData.js](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/token-allowance/token-allowance.js#L263).